utils/lolcat/Makefile

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 OpenWrt.org
```

Comment:
Are you affiliated somehow to OpenWrt? If not, then remove it.

Source:
https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L45

Reply:
Thanks, will fix.